### PR TITLE
Added: Results Page

### PR DESCRIPTION
**Pull request by hussiantabha**
Now users can see results page where what they have selected is right or wrong.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -20,16 +20,19 @@
...
   justify-content: center;
   align-items: center;
   column-gap: 3rem;
+  row-gap: 1.5rem;
 }
 .quiz-card-container {
   display: flex;
   flex-flow: column wrap;
-  border: 1px solid black;
+  border: 1px solid #e5e7eb;
```
Comment (t007rushi) [inline src/App.css]: Color variable

File: src/App.css [hunk 2]
```diff
@@ -20,16 +20,19 @@
...
   border-radius: 0.25rem;
   width: 320px;
```
Comment (t007rushi) [inline src/App.css:38]: Can use px for border and rem for width instead

File: src/components/DisplayResults.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
+import React, { useContext } from "react";
+import { QuizContext } from "../reducers/quizReducer";
+
+const DisplayResults = ({ data }) => {
+  const { quizState, dispatch } = useContext(QuizContext);
+  const filtered = data.filter((item) => item._id <= 5);
+  console.log(filtered);
```
Comment (t007rushi) [inline src/components/DisplayResults.js:7]: Remove consoles

File: src/pages/GadgetQuiz.js [hunk 4]
```diff
@@ -7,9 +7,10 @@ const GadgetQuiz = () => {
   const { quizState, dispatch } = useContext(QuizContext);
   const [gadgetData, setGadgetData] = useState([]);
   const [quizQuestion, setQuizQuestion] = useState(0);
-  const [currentQuestion, setCurrentQuestion] = useState(
-    gadgetData[quizQuestion]
-  );
+  const location = useLocation();
+  // const [currentQuestion, setCurrentQuestion] = useState(
+  //   gadgetData[quizQuestion]
+  // );
```
Comment (t007rushi) [inline src/pages/GadgetQuiz.js]: Remove unwanted code comments

File: src/pages/Results.js [hunk 5]
```diff
@@ -1,13 +1,123 @@
...
-
+import { useLocation, useNavigate } from "react-router-dom";
+import { QuizContext } from "../reducers/quizReducer";
+import DisplayResults from "../components/DisplayResults";
 const Results = () => {
+  const { quizState, dispatch } = useContext(QuizContext);
+  const location = useLocation();
+  const fromPath = location.state.pathname;
+  const navigate = useNavigate();
+  console.log(fromPath);
```
Comment (t007rushi) [inline src/pages/Results.js]: Same here

File: src/pages/Results.js [hunk 6]
```diff
@@ -1,13 +1,123 @@
...
+            : quizState.sportsQuizData
+        }
+      />
     </>
   );
 };
 
 export default Results;
+{
+  /* <section>
```
Comment (t007rushi) [inline src/pages/Results.js]: If it's future scope code then mention it 
Or else remove if done ✅ with it

File: src/pages/FilmQuiz.js [hunk 7]
```diff
@@ -8,36 +8,19 @@ const FilmQuiz = () => {
...
   useEffect(() => {
     dispatch({ type: "filmQuizAnswers", payload: { value: answers } });
     if (quizQuestion === 5) {
-      navigate("/results");
+      navigate("/results", { state: location });
     }
   }, [quizQuestion]);
 
-  console.log(quizState);
+  //console.log(quizState);
```
Comment (vermasavita) [inline src/pages/FilmQuiz.js:23]: remove this comment

File: src/pages/GadgetQuiz.js [hunk 8]
```diff
@@ -7,9 +7,10 @@ const GadgetQuiz = () => {
   const { quizState, dispatch } = useContext(QuizContext);
   const [gadgetData, setGadgetData] = useState([]);
   const [quizQuestion, setQuizQuestion] = useState(0);
-  const [currentQuestion, setCurrentQuestion] = useState(
-    gadgetData[quizQuestion]
-  );
+  const location = useLocation();
+  // const [currentQuestion, setCurrentQuestion] = useState(
```
Comment (vermasavita) [inline src/pages/GadgetQuiz.js]: remove this comment

File: src/pages/Results.js (hunk 5 above)
Comment (vermasavita) [inline src/pages/Results.js]: remove console

File: src/pages/Results.js [hunk 9]
```diff
@@ -1,13 +1,123 @@
...
+  <main className="results-top-container">
+    <div>
+      {fromPath === "/suits-quiz" ? (
+        <h4>
```
Comment (vermasavita) [inline src/pages/Results.js]: You can remove this commented code if not needed

**Review by t007rushi — APPROVED** [review]
Like the ```camelCase``` with clear naming convention
Just Look for consoles and comments 
Else good work keep it up

**Pull request merged**